packages/react/README.md

```diff
@@ -101,3 +119,593 @@ function WidgetDeleter() {
+
+Without any options, `useFindOne` will fetch the record and cause your component to rerender as the fetch happens and when the data or error arrives:
+
+```javascript
+import React from "react";
+import { api } from "../api"; // some file that instantiates @gadget-client/your-app-slug
+
+export const ShowWidgetName = (props: { id: string }) => {
+  const [{ data, fetching, error }, _refetch] = useFindOne(api.widget, props.id);
+
+  if (fetching) return "Loading...";
+  if (error) return `Error loading data: ${error}`;
+  return <span className="widget-name">{data.name}</span>;
+};
+```
+
+`useFindOne` can take options which allow the customization of which fields are returned:
+
+```javascript
+import React from "react";
+import { api } from "../api"; // some file that instantiates @gadget-client/your-app-slug
+
+export const OnlySomeWidgetFields = (props: { id: string }) => {
+  // fetch only the widget id and name fields
+  const [{ data, fetching, error }, _refetch] = useFindOne(api.widget, props.id, { select: { id: true, name: true } });
+
+  return (
+    <span className="widget-name">
+      {data?.id}: {data?.name}
+    </span>
+  );
+};
+```
+
+The `refetch` function returned as the second element can be executed in order to trigger a refetch of the most up to date data from the backend. See [`urql`'s docs](https://formidable.com/open-source/urql/docs/basics/react-preact/#reexecuting-queries) on re-executing queries for more information.
+
+### `useMaybeFindOne(manager: ModelFinder, id: string, options: SingleFinderOptions = {}): [{data, fetching, error}, refetch]`
+
+`useMaybeFindOne` fetches one record from your Gadget database with a given `id`. `useMaybeFindOne(api.widget, 42)` is the React equivalent of `await api.widget.maybeFindOne(42)`. `useMaybeFindOne` will return `data: null` and `error: null` if no record with the given `id` is found in the backend database. `useMaybeFindOne` otherwise behaves identically to `useFindOne`, and accepts the same options.
+
+### `useFindMany(manager: ModelFinder, options: ManyFinderOptions = {}): [{data, fetching, error}, refetch]`
+
+`useFindMany` fetches a page of records from your Gadget database, optionally sorted, filtered, or searched. `useFindMany(api.widget, { ... })` is the React equivalent of `await api.widget.findMany({ ... })`. Options:
+
+- `manager`: The model manager you for the model you want to find a page of records for. The value always comes from an instance of your application's API client instantiated elsewhere. Required. Example: `api.widget`, or `api.shopifyProduct`
+- `options`: Options for making the call to the backend. Not required.
+  - `select`: A list of fields and subfields to select. See the [Select option](#the-select-option) docs. Optional.
+  - `filter`: A list of filters to limit the set of returned records down to. Optional. See the Model Filtering section in your application's API documentation to see the available filters for your models.
```

Comment:
nit: here and any other filter explanation, I'd drop the "down to", since it's redundant.

Another thought: would be really neat to be able to link this to the "Model Filtering" section of some model in the latest/som app you visited on the docs. Not sure if that'd be useful though 🤔